scripts/obfuscator/salt

```diff
@@ -0,0 +1 @@
+m4xQGS9DSKDo!qXDbv5
```

Comment:
What will we do with the fact that the salt will be in the public repository?

Reply:
The repo must be private when we intend to use this feature. We can update the content of this file after hiding it.

Reply:
I think we can use secrets to store the salt https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository

Reply:
@NadzeyaShpakouskaya 
It's required to be the owner of the repo and additional time to figure out. I propose to delay that.

Reply:
I agree, that it requires more action, maybe we can store salt locally and don't push it to the repo.

> The repo must be private when we intend to use this feature. We can update the content of this file after hiding it.

But we can't do this, so we should find a solution for now, we also don't have access to the repo's settings.

Reply:
As discussed, the file with the salt will be stored locally and added to .gitignore for now